src/Database/V5/Bloodhound/Types.hs

```diff
@@ -1394,7 +1460,7 @@ newtype TypeName =
 newtype PercentMatch =
   PercentMatch Double deriving (Eq, Read, Show, Generic, ToJSON, FromJSON, Typeable)
 newtype StopWord =
-  StopWord Text deriving (Eq, Read, Show, Generic, ToJSON, FromJSON, Typeable)
+  StopWord Text deriving (Eq, Read, Show, Generic, ToJSON, FromJSON, Typeable, IsString)
```

Comment:
Don't add `IsString` instances to newtypes, it defeats the purpose of the newtype.